tests/helpers.bash

```diff
@@ -302,10 +303,7 @@ function skip_if_rootless() {
 #  skip_if_no_runtime  #  'buildah run' can't work without a runtime
 ########################
 function skip_if_no_runtime() {
-    # FIXME: if #1964 gets fixed, use 'buildah info' to determine runtime
-    # FIXME: right now we just rely on runc
-    runtime=runc
-    if type -p $runtime &> /dev/null; then
+    if type -p ${OCI} &> /dev/null; then
```

Comment:
Related to the above (re: `command -v`): this should include `-n "${OCI}" && ...`